.gitignore

```diff
@@ -128,3 +128,4 @@ kubernetes.tar.gz
 
 # test coverage file
 coverage.txt
+/apis/
```

Comment:
If we removed the verify gencode, why is this folder ignored?

Reply:
in case `hack/update-codegen.sh` is run by user mistakenly. We can also remove it.

Reply:
I understand what you mean, I think there is no problem with this PR at present. `hack/update-codegen.sh` is recommended to keep.

Reply:
ok, thanks